docs/4_1_co-simulation_math.adoc

```diff
@@ -81,15 +81,12 @@ This section contains a formal mathematical model of a Co-Simulation FMU.
 The following fundamental assumptions are made:
 
 The slave simulators are seen by the master simulator as purely sampled-data systems.
-Such a sampled-data system can be:
 
-- A real sampled-data system (so a sampled discrete controller; the <<input,`inputs`>> and <<output,`outputs`>> can be of type Real, Integer, Boolean, String, or enumeration.
-Variables of this type are defined with <<variability>> = <<discrete>>; the smallest sample period as accessible by the outside of the FMU is defined by attribute `stepSize` in element `DefaultExperiment`).
-
-- A hybrid ODE that is integrated between communication points (known as "sampled access to time continuous systems") where internal events may occur and be handled, but events are not visible from the outside of the FMU.
+Such a sampled-data system consists typically of a hybrid ODE that is integrated between communication points (known as "sampled access to time continuous systems") where internal events may occur and be handled, but events are not visible from the outside of the FMU.
 It is assumed here that all <<input,`inputs`>> and all <<output,`outputs`>> of this hybrid ODE are Real signals (defined with <<variability>> = <<continuous>>),
 
-- A combination of the systems above.
+_[ In FMI 2.0, Co-Simulation FMUs were also used for real sampled-data systems (so a sampled discrete controller; the <<input,`inputs`>> and <<output,`outputs`>> could be of type Real, Integer, Boolean, String, or enumeration with <<variability>> = <<discrete>>._
+_However, in FMI 3.0, Hybrid Co-Simulation (HCS) and Scheduled Execution Simulation (SES) are intended for this use-case.]_
```

Comment:
Frankly, I don't understand the point of this discussion. Would it be helpful if we change inline 89 "are intended" into "are more suitable".

Reply:
We explain the use cases of the different interface types at another place. 
(see pull request https://github.com/modelica/fmi-standard/pull/728)
I would not repeat that here. The issue #434 is solved in my view with the updates Christian provided.